src/server/status/status.modal.js

```diff
@@ -21,32 +25,48 @@
                 showTimeout = undefined;
             }
             if(ServerStatusService.hasErrors()){
-                ServerStatusModal.open();
+                ServerStatusModal.open('error');
             } else {
                 ServerStatusModal.close();
             }
         });
     }
 
-    statusModal.$inject = ['$q', '$compile', '$rootScope', 'BootstrapDialog', 'ServerStatusService'];
-    function statusModal ($q, $compile, $rootScope, BootstrapDialog, ServerStatusService) {
-        var modal = new BootstrapDialog({
-            title: $compile('<span>{{ "server.welcome.startup" | translate }}</span>')($rootScope),
-            message: $compile('<motech-server-status />')($rootScope),
-            buttons: [],
-            closable: false
-        });
+    statusModal.$inject = ['$compile', '$rootScope', 'BootstrapDialog'];
+    function statusModal ($compile, $rootScope, BootstrapDialog) {
+        var title = 'Welcome Startup',
```

Comment:
@wstrzelczyk I think this is arguable, but I feel that discussing this would be good -- why are we not translating these strings?

I feel that even if these are translations from the client side, we should still be translating...

Is there a reason why you chose not to translate? Was there a bug that was blocking translation?

Perhaps translating here could be a separate ticket? I would be ok with that if the potential fix is outside of the scope of this ticket